users: explain user_bind_dn field (#93)
NethServer/nethsecurity-docs: migration.rst

If you have performed an in-place migration, you must :ref:`register the system <subscription-section>` again.
This step is not necessary if you have performed a migration with the exported archive method.

> If your using a remote LDAP or Active Directory server to authenticate OpenVPN Road Warrior clients, make sure that the remote server is 

Reviewer: ```suggestion
When using a remote LDAP or Active Directory server to authenticate OpenVPN Road Warrior clients, make sure that the remote server is 
```
